bin/rpm.go

```diff
@@ -238,44 +244,39 @@ func doClientRPM() error {
 	if input == "" {
 		input, err = os.Executable()
 		if err != nil {
-			return fmt.Errorf("Unable to open executable: %w", err)
+			return fmt.Errorf("Unable to find executable: %w", err)
 		}
 	}
 
-	fd, err := os.Open(input)
-	if err != nil {
-		return fmt.Errorf("Unable to open executable: %w", err)
-	}
-	defer fd.Close()
-
-	header := make([]byte, 4)
-	_, err = fd.Read(header)
+	e, err := elf.Open(input)
 	if err != nil {
-		return fmt.Errorf("Unable to open executable: %w", err)
+		return fmt.Errorf("Unable to parse ELF executable: %w", err)
 	}
 
-	if binary.LittleEndian.Uint32(header) != 0x464c457f {
```

Comment:
Maybe add that error message back again to help people who dont know about the flag